src/mono/wasm/runtime/marshal.ts

```diff
@@ -18,14 +18,17 @@ export const bound_js_function_symbol = Symbol.for("wasm bound_js_function");
 export const imported_js_function_symbol = Symbol.for("wasm imported_js_function");
 export const proxy_debug_symbol = Symbol.for("wasm proxy_debug");
 
-export const JavaScriptMarshalerArgSize = 16;
+export const JavaScriptMarshalerArgSize = 32;
```

Comment:
is there a specific reason why we doubled this to 32 instead of just growing it by 4 bytes?

Reply:
Because of how C# "packing" works. When you make `Span<JSMarshalerArgument>` size `20` would be rounded to `32` anyway.